obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -11,7 +11,12 @@ RUN yum install -y \
                    patch \
                    tar \
                    wget \
-                   which
+                   which \
+                   libXext \
+                   libXrender \
+                   libSM \
+                   tk \
```

Comment:
I think it should not be there, because the current mpl packages explicitly depend on a `tk` package.